data-es7.js

```diff
@@ -1233,7 +1275,7 @@ exports.tests = [
       */},
       res: {
         babel:      true,
-        edge:       true,
+        edge12:       true,
         firefox31:  true,
```

Comment:
done and sorry, hope its okay now. also fixed some that were unaligned the first place.